src/serac/numerics/equation_solver.cpp

```diff
@@ -197,10 +204,10 @@ class NewtonSolver : public mfem::NewtonSolver {
     final_iter = it;
     final_norm = norm;
 
-    if (print_options.summary || (!converged && print_options.warnings) || print_options.first_and_last) {
-      mfem::out << "Newton: Number of iterations: " << final_iter << '\n' << "   ||r|| = " << final_norm << '\n';
+    if (print_level == 1) {
```

Comment:
if the print level is 2, wouldn't you want to print this section too? there are a couple other sections like this

`if (print_level <= 1)` ?